WMFComponents/Sources/WMFComponents/Components/Article Tabs/WMFArticleTabsViewModel.swift

```diff
@@ -89,12 +77,29 @@ public class WMFArticleTabsViewModel: NSObject, ObservableObject {
     private func updateShouldShowSuggestions() {
         shouldShowSuggestions = dataController.shouldShowMoreDynamicTabsV2 &&
         !dataController.userHasHiddenArticleSuggestionsTabs
-}
+    }
     
     public func refreshShouldShowSuggestionsFromDataController() {
         updateShouldShowSuggestions()
     }
     
+    @MainActor
+    private func updateHasMultipleTabs() {
+        var count = 0
+        if articleTabs.isEmpty {
+            hasMultipleTabs = false
+            return
+        }
+        for tab in articleTabs where !tab.isMain {
+            count += 1
+            if count > 2 {
```

Comment:
FYI we should change this back to >= 2 non-Main tabs - I hashed out conflicts with Product and that was the agreed upon value for showing the "Because you read" module.